pkg/controller/podautoscaler/horizontal.go

```diff
@@ -39,6 +39,8 @@ const (
 	// TODO: make it a flag or HPA spec element.
 	tolerance = 0.1
 
+	defaultTargetCPUUtilizationPercentage = 80
```

Comment:
Nope. It was set by default to 80 in extensions/v1beta1 in [defaults.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/extensions/v1beta1/defaults.go#L117). In autoscaling/v1 default is applied in controller rather than API level which will allow us to change it in the future without making a breaking change in API.